scylla-macros/src/into_user_type.rs

```diff
@@ -14,20 +14,20 @@ pub fn into_user_type_derive(tokens_input: TokenStream) -> TokenStream {
         let field_name = &field.ident;
 
         quote_spanned! {field.span() =>
-            <_ as Value>::serialize(&self.#field_name, buf) ?;
+            <_ as scylla::frame::value::Value>::serialize(&self.#field_name, buf) ?;
         }
     });
 
     let generated = quote! {
         impl #impl_generics scylla::frame::value::Value for #struct_name #ty_generics #where_clause {
-            fn serialize(&self, buf: &mut Vec<u8>) -> std::result::Result<(), scylla::frame::value::ValueTooBig> {
+            fn serialize(&self, buf: &mut ::std::vec::Vec<::core::primitive::u8>) -> ::std::result::Result<(), scylla::frame::value::ValueTooBig> {
                 use scylla::frame::value::{Value, ValueTooBig};
                 use scylla::macros::BufMut;
                 use ::std::convert::TryInto;
```

Comment:
Would importing the primitive types, just like `BufMut/TryInto/etc.` here work?

```rust
use ::std::primitive::{i32, usize};
```

Reply:
Yes, though I used `core` instead of `std` for no particular reason